style_editor.html

```diff
@@ -9666,6 +9701,10 @@
 }
 
 function ActivateTab(tab) {
+  if (tab === "arg_string") {
```

Comment:
If safeGuardInputs is run on page init, is this needed?  Will it cause multiple copies of the event listeners to get attached to existing inputs?

Reply:
In testing, the inputs on the ArgString tab are not governed unless this is applied upon clicking the tab.

Reply:
Right because they're dynamically added.  I wonder if maybe safeguardInputs should be updated to take a selector as a string arg.  Because every single time someone leaves and returns to the argstring tab, this will get applied to all of the static elements as well.  As a result, you might get 20 triggers on a single click of the "Power" button.  Which probably doesn't need it at all.

Reply:
Existing inputs were all `type=input`. Apparently there is no type 'input' type for an input element, but it's been working fine.
SOoto make this more cohesive, I changed them to type=text, then filtered the safeguardInputs to only those inputs.

Reply:
That seems solid!